src/Haskoin/Address/Base58.hs

```diff
@@ -38,84 +36,91 @@ import Haskoin.Crypto.Hash
 import Haskoin.Util
 import Numeric (readInt, showIntAtBase)
 
+
 -- | 'Base58' classic Bitcoin address format.
 type Base58 = Text
 
+
 -- | Symbols for Base58 encoding.
 b58Data :: ByteString
 b58Data = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
 
+
 b58Array :: Array Int Word8
 b58Array = listArray (0, 57) (BS.unpack b58Data)
 
+
 b58InvArray :: Array Word8 (Maybe Int)
 b58InvArray = listArray (minBound, maxBound) (repeat Nothing) // map swap (assocs b58Array)
-  where
-    swap (i, c) = (c, Just i)
+    where
+        swap (i, c) = (c, Just i)
 
-{- | Convert a number less than or equal to provided integer into a 'Base58'
- character.
--}
+
+-- | Convert a number less than or equal to provided integer into a 'Base58'
+-- character.
 b58 :: Int -> Word8
 b58 = (b58Array !)
 
+
 -- | Convert a 'Base58' character into the number it represents.
 b58' :: Word8 -> Maybe Int
 b58' = (b58InvArray !)
 
-{- | Encode an arbitrary-length 'Integer' into a 'Base58' string. Leading zeroes
- will not be part of the resulting string.
--}
+
+-- | Encode an arbitrary-length 'Integer' into a 'Base58' string. Leading zeroes
+-- will not be part of the resulting string.
 encodeBase58I :: Integer -> Base58
 encodeBase58I i = cs $ showIntAtBase 58 (chr . fromIntegral . b58) i ""
 
+
 -- | Decode a 'Base58' string into an arbitrary-length 'Integer'.
 decodeBase58I :: Base58 -> Maybe Integer
 decodeBase58I s =
     case go of
         Just (r, []) -> Just r
         _ -> Nothing
-  where
-    p = isJust . b58' . fromIntegral . ord
-    f = fromMaybe e . b58' . fromIntegral . ord
-    go = listToMaybe $ readInt 58 p f (cs s)
-    e = error "Could not decode base58"
-
-{- | Encode an arbitrary 'ByteString' into a its 'Base58' representation,
- preserving leading zeroes.
--}
+    where
+        p = isJust . b58' . fromIntegral . ord
+        f = fromMaybe e . b58' . fromIntegral . ord
+        go = listToMaybe $ readInt 58 p f (cs s)
+        e = error "Could not decode base58"
+
+
+-- | Encode an arbitrary 'ByteString' into a its 'Base58' representation,
+-- preserving leading zeroes.
```

Comment:
I like this comment style better.  Good call.

Reply:
ha, my formatter did it for me...